Corgibytes.Freshli.Cli/Functionality/Cache.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using ServiceStack;
+
+namespace Corgibytes.Freshli.Cli.Functionality
+{
+    [Index(nameof(Key), IsUnique = true)]
+    public class CachedProperty
+    {
+        public int Id { get; set; }
+        public string Key { get; set; }
+        public string Value { get; set; }
+    }
+
+    class CacheContext : DbContext
+    {
+        public static readonly string cacheDirEnvVariable = "FRESHLI_CACHE_DIR";
+        public static readonly DirectoryInfo cacheDirDefault = new DirectoryInfo(Environment.GetEnvironmentVariable("HOME") + "/.freshli");
+        public static readonly string cacheDbName = "freshli.db";
+        public string DbPath { get; }
+
+        public static string CacheDir
+        {
+            get
+            {
+                string cacheDir = Environment.GetEnvironmentVariable(CacheContext.cacheDirEnvVariable);
+                return cacheDir.IsNullOrEmpty() ? CacheContext.cacheDirDefault.ToString() : cacheDir;
+            }
+        }
+
+        public DbSet<CachedProperty> CachedProperties { get; set; }
+
+        public CacheContext()
+        {
+            DbPath = System.IO.Path.Join(CacheDir, cacheDbName);
```

Comment:
Looks like this isn't utilizing the global option that is provided on the command line. Am I missing something?

Reply:
It IS using it, but I store the overridden path on an environment variable, so external calls to 'dotnet ef' are not broken. See line 64 for where I set that.